docs/manual-CN/2.query-language/4.statement-syntax/1.data-definition-statements/create-tag-edge-syntax.md

```diff
@@ -97,4 +97,4 @@ nebula> CREATE EDGE garbage (thrown timestamp, temperature int)
    TTL_DURATION = -2, TTL_COL = thrown,
    TTL_DURATION = 10, TTL_COL = thrown
    --  语法合法，但不推荐。数据将在 thrown + 10 后失效
-``` -->
```

Comment:
You can specify multiple `TTL_DURATION `or `TTL_COL`,   but only the last one is valid.
So this usage like this is not recommended.